Webpack errors out on images

I cloned this repo and tried adding an image into `src/assets/img` and referencing in the Page1.html like so:

    <img src="assets/img/lego.jpg">

However, webpack chokes on this with a "Module not found" error and prevents any of the tests from running:

    ERROR in ./src/pages/page1/page1.html
    Module not found: Error: Can't resolve './assets/img/lego.jpg' in 'C:\dev\ionic-unit-testing-example-master\src\pages\page1'
     @ ./src/pages/page1/page1.html 1:405-437
     @ ./src/pages/page1/page1.ts
     @ ./src/pages/page1/page1.spec.ts
     @ ./src \.spec\.ts
     @ ./test-config/karma-test-shim.js
 

I assume this requires a modification to the webpack.config.js to get it to look in the correct location but I am not that familiar with webpack so I'm not sure the fix. I did note that images are loaded using "null_loader" which I understand just loads a null module, in which case I am not sure why webpack cares about where the image is.

Anyone know what needs to be changed to prevent this problem?

--- a/test-config/webpack.test.js
+++ b/test-config/webpack.test.js
@@ -20,7 +20,7 @@ module.exports = {
       },
       {
         test: /\.html$/,
-        loader: 'html-loader'
+        loader: 'html-loader?attrs=false'
       },
       {
         test: /\.(png|jpe?g|gif|svg|woff|woff2|ttf|eot|ico)$/,

